package.json

```
@@ -27,5 +29,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^13.13.0"
   }
-}
+}
```

```suggestion
}

```

É uma convenção deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como GitHub não exibem o ícone de _No newline at end of file_.